### ISSUE TITLE
Weird Result With Custom Mask

### ISSUE THREAD

**Issue by sshh12** [JiahuiYu/generative_inpainting#404]
When using the pretrained model with a custom mask, I get weird results:

![case1_raw](https://user-images.githubusercontent.com/6625384/73147276-643c0980-407c-11ea-86c2-4b58f57c0c75.png)

<img width="253" alt="PaintDotNet_2020-01-26_20-42-01" src="https://user-images.githubusercontent.com/6625384/73147259-525a6680-407c-11ea-90b9-2f83e31c49a4.png">

![out](https://user-images.githubusercontent.com/6625384/73147268-58504780-407c-11ea-9bd6-77f86bf7718f.png)

## Steps
1. `git clone https://github.com/JiahuiYu/generative_inpainting.git`
2. Download provided Places2 model to `model_logs\release_places2_256`
3. Install `tensorflow==1.7.0` and `git+https://github.com/JiahuiYu/neuralgym`
4. `python test.py --image examples/places2/case1_raw.png --mask examples/places2/case1_mask.png --output out.png --checkpoint_dir model_logs/release_places2_256`
5. `python test.py --image examples/places2/case1_raw.png --mask custom-mask.png --output out.png --checkpoint_dir model_logs/release_places2_256`

Step 4 is pretty much the command on the README and it works (result matches output example) but Step 5 generates the crazy output above. The `custom-mask.png` is the correct size with only 3 pure-white circles drawn on it.

Also not sure how this is even possible given https://github.com/JiahuiYu/generative_inpainting/blob/cfc1d41a882dbbe7c2c35446c8e49b81b7ed51ff/inpaint_model.py#L296

Raw mask file: https://drive.google.com/open?id=1jWu2zXR7I2WOBQLTQEWf-g94Em8Z2CxK

**Comment by sshh12** [comment]
Pip:
```
neuralgym==0.0.1
numpy==1.18.1
opencv-python==4.1.2.30
tensorboard==1.7.0
tensorflow-gpu==1.7.0
Pillow==7.0.0
```
Python: `Python 3.5.6 :: Anaconda, Inc.`
OS: `Windows 10`
CUDA: `10.1`

**Comment by sshh12** [comment]
Ah figured it out, my editor was saving the empty space as (R255,G255,B255,A0) but OpenCV expects it to be (R0,G0,B0,A0). Filling in the empty space with solid black fixes it.

**Comment by redaghanem** [comment]
@sshh12  please how you figure problem of a mask I need to test this image in my own mask

how can i create a correct mask

![input2](https://user-images.githubusercontent.com/43389843/73214194-607eb480-415a-11ea-98f8-a9fc974aa728.png)
![input3](https://user-images.githubusercontent.com/43389843/73214195-607eb480-415a-11ea-8231-fcd9b58fd156.png)
![mask3](https://user-images.githubusercontent.com/43389843/73214196-607eb480-415a-11ea-9fe4-b72fed714e17.png)

**Comment by redaghanem** [comment]
@sshh12 
when i invert mask like this

![mask3](https://user-images.githubusercontent.com/43389843/73215155-2b736180-415c-11ea-8581-4635fe007652.png)

i get this 


![774d59bd-6bd7-4e19-a774-544d078c17b9](https://user-images.githubusercontent.com/43389843/73215179-35956000-415c-11ea-8bd8-e9a5760cc6c0.png)


please tell me how can i make my manula mask for any image to test it

**Comment by sshh12** [comment]
Invert your example mask and ensure there is no transparency.

To clarify: When I say "empty space" I mean the part of the image that is not modified. So the do-not-touch part of the image should be black and the in-paint part should be white.

**Comment by redaghanem** [comment]
Thanks for clarification i will try this, I wish to generate a good result
